lib/delocalize/parsers/date_time.rb

```diff
@@ -51,7 +51,7 @@ def default_parse(datetime, type)
 
         today = Date.current
         parsed = Date._parse(datetime)
-        return if parsed.empty? # the datetime value is invalid
+        raise ArgumentError, 'invalid date' if parsed.empty? # the datetime value is invalid
```

Comment:
I think this should be `raise ArgumentError, "invalid date: #{parsed}" if parsed.empty? # the datetime value is invalid`. I know that this differs slightly from Ruby's default handling but honestly I think Ruby itself should show the invalid input as well.

What do you think?